pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -238,6 +243,11 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 				err = f(r.client, instance)
 				if err != nil {
 					reqLogger.Error(err, "Failed to manually delete objects")
+					r.recorder.Event(instance, corev1.EventTypeWarning, ErrHCOUninstall, uninstallErrorMsg)
```

Comment:
my only concern with this approach is that there are reasons that we could fail that have nothing to do with CDI or KubeVirt's validating webhooks.

Reply:
by the way we are neither trying to explicitly delete kubevirt but just CDI

Reply:
to keep it simple, let's try to be as generic as possible and consume the error message we get from the component without trying to parse the content